doc/releases/release-notes-3.4.rst

```diff
@@ -277,6 +277,25 @@ Build system and infrastructure
   see :ref:`West extending signing <west-extending-signing>` for further
   details.
 
+* west sign command internal logic no longer uses label property of "fixed-partition" compatible
+  nodes to identify application image slots, and uses DTS labels instead. This means that
+  if custom partition layout is used with MCUboot partitions given label property, with image
+  identifier, should be now given proper DTS node label. Label property does not have to be
```

Comment:
@de-nordic This hasn't been addressed, and I find the sentence really hard to understand. Would the following work?

```
* `west sign` command no longer uses label property of "fixed-partition" compatible
  nodes to identify application image slots, and uses DT labels instead. This means
  that when using a custom partition layout with MCUboot partitions, you now need to
  assign a DT node label to each slot previously assigned via the `label` property.
  The `label` property doesn't need to be removed, but it will no longer be used.
```

Thanks!